etc/jest/config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  rootDir: '../../src',
+  transform: {
+    '^.+\\.(t|j)sx?$': ['@swc/jest'],
+    '^.+\\.css$': '<rootDir>/../etc/jest/cssTransform.js',
+    '^(?!.*\\.(js|jsx|mjs|cjs|ts|tsx|css|json)$)': '<rootDir>/../etc/jest/fileTransform.js',
+  },
+  transformIgnorePatterns: [
+    '[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs|cjs|ts|tsx)$',
+    '^.+\\.module\\.(css|sass|scss)$',
+  ],
+  setupFilesAfterEnv: ['<rootDir>/setupTests.ts'],
+};
```

Comment:
This could be defined in the package json file?

https://jestjs.io/docs/en/configuration

Reply:
Yes it could, but we have most of the configs in the `/etc` folder.

Reply:
And given you need other files like the _file/css transformers_, I thought it was looking better all in the `/etc/jest` folder